app/controllers/reports_controller.rb

```diff
@@ -8,6 +8,7 @@ def show
     vaccinations =
       policy_scope(VaccinationRecord)
         .administered
+        .recorded
```

Comment:
Makes me wonder if these shouldn't be default scopes ... as imperfect as those can be.